rmw_cyclonedds_cpp/src/rmw_node.cpp

```diff
@@ -309,10 +314,23 @@ struct CddsSubscription : CddsEntity
   dds_entity_t rdcondh;
 };
 
+struct client_service_id_t
+{
+  // strangely, the writer_guid in a request id is smaller than the rmw_gid_t
+  // cpplint erroneously complains that
+  //
+  //   sizeof((reinterpret_cast<rmw_request_id_t *>(0))->writer_guid)
+  //
+  // is not a constant, and the 16 is a hard-coded magic number in
+  // rmw_request_id_t ...
```

Comment:
I don't fully follow the comment, but you can also suppress a cpplint warning by using `problematic line of code;  // NOLINT`.

Reply:
There are actually two comments with bad formatting: one that I put in originally: "strangely, the writer_guid in a request id is smaller than the rmw_gid_t". I find it weird that the writer_guid in a `rmw_request_t` is 16 bytes whereas a `rmw_gid_t` has 24 bytes.

The second bit is about lint. I didn't know about `// NOLINT`, but that's a better solution than a hard-coded constant. Thanks!